# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/telemetry/Histograms.json
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 toolkit/components/telemetry/Histograms.json
--- a/toolkit/components/telemetry/Histograms.json	Fri Jan 27 19:06:16 2023 +0000
+++ b/toolkit/components/telemetry/Histograms.json	Fri Jan 27 21:11:42 2023 +0200
@@ -1791,7 +1791,6 @@
       "alpha_y_bpc_mismatch",
       "ispe_mismatch",
       "invalid_cicp",
-      "no_samples",
       "invalid_parse_status",
       "missing_brand",
       "ftyp_not_first",
@@ -1806,8 +1805,7 @@
       "item_type_missing",
       "construction_method",
       "item_loc_not_found",
-      "no_item_data_box",
-      "uncategorized"
+      "no_item_data_box"
     ],
     "description": "Decode result of AVIF image",
     "bug_numbers": [1670827]